### ISSUE TITLE
Error on loopback in Img2Img.

### ISSUE THREAD

**Issue by MarcusMcNut** [hlky/stable-diffusion#76]
I tried to setup a batch count of 8 for batches of 3.  I'm using a 512x512 image, and I'm getting the following error.

`einsum(): operands do not broadcast with remapped shapes [original->remapped]: [48, 4096, 40]->[48, 4096, 1, 40] [16, 77, 40]->[16, 1, 77, 40]`  I've tried this multiple times with different sized images and always have the same error.

**Comment by hlky** [comment]
This is on stable branch, correct?

Either way, Loopback is not a particularly useful feature, so for now I would suggest doing it manually which will provide better results see [here](https://github.com/hlky/stable-diffusion-webui#loopback)

**Comment by MarcusMcNut** [comment]
> This is on stable branch, correct?
> 
> Either way, Loopback is not a particularly useful feature, so for now I would suggest doing it manually which will provide better results see [here](https://github.com/hlky/stable-diffusion-webui#loopback)

Yep this time around I am on the stable branch.  👍 I'll read the link.  Thanks.

**Comment by hlky** [comment]
Thanks, I'm going to move this to the dev repo since it will be broken there too, the feature might be removed at some point, or probably extracted to its own tab away from the main img2img
